### PR TITLE
[MINOR] - Vertx and Netty minor version bumps

### PR DESCRIPTION
**Pull request by Gabriel-Trintinalia**
## PR Description

Updated dependencies:
- Vert.x: `4.5.4` -> `4.5.8`
- Netty: `4.1.104.Final` -> `4.1.110.Final`

### CODE REVIEW

File: gradle/verification-metadata.xml [hunk 1]
```diff
@@ -1664,267 +1664,207 @@
...
+            <sha256 value="55b2458011527d94abc868086afd039cd00cc3a547e7322569e0fb4f906d9d80" origin="Generated by Gradle"/>
          </artifact>
-         <artifact name="netty-handler-4.1.107.Final.pom">
-            <sha256 value="c6c59fd6f0f70a043a50a5826de7ad68dbb95d341e03bd1bbe179038e48b845e" origin="Generated by Gradle"/>
+         <artifact name="netty-handler-4.1.108.Final.pom">
+            <sha256 value="e621cafa98907ed01e8a713ff4aaa650943591b023b4f6ee249e4deb9f8d566f" origin="Generated by Gradle"/>
          </artifact>
       </component>
       <component group="io.netty" name="netty-handler-proxy" version="4.1.100.Final">
          <artifact name="netty-handler-proxy-4.1.100.Final.pom">
```
Comment (pinges) [inline gradle/verification-metadata.xml:1816]: should that be 4.1.108 as well?

File: gradle/verification-metadata.xml [hunk 2]
```diff
@@ -1664,267 +1664,207 @@
...
+         <artifact name="netty-handler-ssl-ocsp-4.1.108.Final.jar">
+            <sha256 value="2e9219f324fbc3ea345667cf43ae58dd0841cc94cce116fbd724aa9bc47d5f41" origin="Generated by Gradle"/>
          </artifact>
-         <artifact name="netty-handler-ssl-ocsp-4.1.104.Final.pom">
-            <sha256 value="aee924520525966b3481ebfee6ee95c3a206d23f07f69270557a9dd2f73dead6" origin="Generated by Gradle"/>
+         <artifact name="netty-handler-ssl-ocsp-4.1.108.Final.pom">
+            <sha256 value="f65a5d25178de89e666e69350251561988b1593bd4a5e867783f734efe04be13" origin="Generated by Gradle"/>
          </artifact>
       </component>
       <component group="io.netty" name="netty-parent" version="4.1.100.Final">
```
Comment (pinges) [inline gradle/verification-metadata.xml:1836]: s.a.

File: gradle/verification-metadata.xml [hunk 3]
```diff
@@ -1664,267 +1664,207 @@
...
-         <artifact name="netty-resolver-dns-classes-macos-4.1.104.Final.jar">
-            <sha256 value="5d45c0fb39d911e2470de5ed6480a23332f454838923ffceb0cf1b4002ecd713" origin="Generated by Gradle"/>
-         </artifact>
-         <artifact name="netty-resolver-dns-classes-macos-4.1.104.Final.pom">
-            <sha256 value="cc6a0c226b1fce0fc56674bc0c021b6512379025bb0cb0608be6a38e370dcd4f" origin="Generated by Gradle"/>
+         <artifact name="netty-resolver-dns-classes-macos-4.1.108.Final.pom">
+            <sha256 value="682a939501e89108d4a7ec45dc2962b853da00127dfb07542de966ab50839782" origin="Generated by Gradle"/>
          </artifact>
       </component>
       <component group="io.netty" name="netty-resolver-dns-native-macos" version="4.1.104.Final">
```
Comment (pinges) [inline gradle/verification-metadata.xml:1870]: s.a.

File: gradle/verification-metadata.xml [hunk 4]
```diff
@@ -1972,6 +1923,14 @@
...
       </component>
+      <component group="io.netty" name="netty-transport" version="4.1.108.Final">
+         <artifact name="netty-transport-4.1.108.Final.jar">
+            <sha256 value="fef2ec66fe01aa89734db40f292676719da3985786512fc31a9efe1ca4d2e0ff" origin="Generated by Gradle"/>
+         </artifact>
+         <artifact name="netty-transport-4.1.108.Final.pom">
+            <sha256 value="453e9a0de74529d0551e53eca11c91d561c55e1665ae40637193b84567692cd6" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
       <component group="io.netty" name="netty-transport-classes-epoll" version="4.1.104.Final">
```
Comment (pinges) [inline gradle/verification-metadata.xml:1934]: s.a.

File: gradle/verification-metadata.xml [hunk 5]
```diff
@@ -1972,6 +1923,14 @@
...
          <artifact name="netty-transport-classes-epoll-4.1.104.Final.jar">
```
Comment (pinges) [inline gradle/verification-metadata.xml:1935]: s.a.

File: gradle/verification-metadata.xml [hunk 6]
```diff
@@ -1980,6 +1939,14 @@
...
       </component>
+      <component group="io.netty" name="netty-transport-classes-epoll" version="4.1.108.Final">
+         <artifact name="netty-transport-classes-epoll-4.1.108.Final.jar">
+            <sha256 value="5959715036c1dfc1b5a41a6b8518762f43b99c9f6f45e5c80543550cb4773c88" origin="Generated by Gradle"/>
+         </artifact>
+         <artifact name="netty-transport-classes-epoll-4.1.108.Final.pom">
+            <sha256 value="404ae90fb49e933b4bcd9d892fb7f9073329645b36c9d469929e4da5f1dc61ab" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
       <component group="io.netty" name="netty-transport-classes-kqueue" version="4.1.104.Final">
```
Comment (pinges) [inline gradle/verification-metadata.xml:1950]: s.a.

File: gradle/verification-metadata.xml [hunk 7]
```diff
@@ -1988,6 +1955,14 @@
...
       </component>
+      <component group="io.netty" name="netty-transport-classes-kqueue" version="4.1.108.Final">
+         <artifact name="netty-transport-classes-kqueue-4.1.108.Final.jar">
+            <sha256 value="85a4e4ec4091c941f9736eb8986443ec2dde3e096a6723fd895289d28b7c3811" origin="Generated by Gradle"/>
+         </artifact>
+         <artifact name="netty-transport-classes-kqueue-4.1.108.Final.pom">
+            <sha256 value="488f3d56a04f76fd832330486969575d29410fa2c30c6e1dbca5187ef47360c7" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
       <component group="io.netty" name="netty-transport-native-epoll" version="4.1.104.Final">
```
Comment (pinges) [inline gradle/verification-metadata.xml:1966]: s.a.

File: gradle/verification-metadata.xml [hunk 8]
```diff
@@ -2005,6 +1980,23 @@
...
+            <sha256 value="452b8f0ae41fab3abb55d7e591be7186068b734c391ea25e8f9af067fd06e83f" origin="Generated by Gradle"/>
+         </artifact>
+         <artifact name="netty-transport-native-epoll-4.1.108.Final.jar">
+            <sha256 value="a825c5320c545360c91249ae8718c56b33135abc8943057479647b4e018dab31" origin="Generated by Gradle"/>
+         </artifact>
+         <artifact name="netty-transport-native-epoll-4.1.108.Final.pom">
+            <sha256 value="382db1962bfaf99339bdfe2dfb8daa2cdfe32c16a589680ef1527ffdbe53de36" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
       <component group="io.netty" name="netty-transport-native-kqueue" version="4.1.104.Final">
```
Comment (pinges) [inline gradle/verification-metadata.xml:2000]: s.a.

**Review by pinges — CHANGES_REQUESTED** [review]
I can see a lot of the old version (104) and even (100). Please check these.

File: gradle/versions.gradle [hunk 9]
```diff
@@ -116,7 +116,7 @@ dependencyManagement {
       entry 'tuweni-units'
     }
 
-    dependencySet(group: 'io.vertx', version: '4.5.4') {
+    dependencySet(group: 'io.vertx', version: '4.5.6') {
```
Comment (jframe) [inline gradle/versions.gradle:119]: Should use Vertx 4.5.8 as that is the latest version

**Review by pinges — APPROVED** [review]
LGTM

**Pull request merged**